README.rst

```diff
@@ -44,6 +43,9 @@ verified. For more information, check
 .. _Amazon's instructions on verifying email addresses:
   http://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-email-addresses.html
 
+.. _Ubuntu:
```

Comment:
It would be better to move it just under Red Hat to keep the same order as in the text.

Reply:
Fixed.